.rubocop_todo.yml

```diff
@@ -38,7 +38,7 @@ Metrics/MethodLength:
 # Offense count: 1
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 307
+  Max: 350
```

Comment:
This was hardcoded to the current size which does not make sense I think so I bumped it a lot so that others don't get lost like I was

Reply:
You can regenerate the configuration with `rubocop --auto-gen-config`. Those metrics cops can be a bit annoying though, we should consider disabling some of them or setting real limits we want to enforce.

Reply:
Is it okay to do what I did to avoid this for every future pr adding a new resource?

Reply:
Yeah, it's fine for now. I'm guessing the largest module is `Stripe::Util`. We can look into fixing these metrics later.